packages/ColorPicker/src/color-picker-option.scss

```diff
@@ -15,9 +15,23 @@
 
   /* Selected */
   --ids-color-picker-option-border-color-selected: #{tokens.$grey-400};
-  
-  /* Icon */
-  --ids-color-picker-option-icon-color: #{tokens.$samoyed};
+
+}
+
+[data-brand="workleap"] {
+  /* Default */
+  --ids-color-picker-option-border-radius: var(--hop-shape-circle);
+  --ids-color-picker-option-border-size: 0.09rem;
```

Comment:
1.44px seems arbitrary.

Reply:
Well it called for 1.5px so I rounded 0.09375rem. That's just way too long. Plus browsers don't end up computing the exact amount anyway.

Reply:
True in that case we might need to ask design to stop using .5px for borders 😄

Reply:
haha maybe... I guess I'll keep it for now?